packages/instant/webpack.config.js

```diff
@@ -1,42 +1,95 @@
 const path = require('path');
+const webpack = require('webpack');
+const RollbarSourceMapPlugin = require('rollbar-sourcemap-webpack-plugin');
+const childProcess = require('child_process');
 const ip = require('ip');
+
 // The common js bundle (not this one) is built using tsc.
 // The umd bundle (this one) has a different entrypoint.
-const outputPath = process.env.WEBPACK_OUTPUT_PATH || 'umd';
-const config = {
-    entry: {
-        instant: './src/index.umd.ts',
-    },
-    output: {
-        filename: '[name].js',
-        path: path.resolve(__dirname, outputPath),
-        library: 'zeroExInstant',
-        libraryTarget: 'umd',
-    },
-    devtool: 'source-map',
-    resolve: {
-        extensions: ['.js', '.json', '.ts', '.tsx'],
-    },
-    module: {
-        rules: [
-            {
-                test: /\.(ts|tsx)$/,
-                loader: 'awesome-typescript-loader',
+const GIT_SHA = childProcess
+    .execSync('git rev-parse HEAD')
+    .toString()
+    .trim();
+
+const ROLLBAR_ACCESS_KEY_ENV_NAME = 'ZEROEX_INSTANT_ROLLBAR_POST_ACCESS_KEY';
+function getPlugins(webpackEnv) {
+    const basePlugins = [
+        new webpack.DefinePlugin({
+            'process.env': {
```

Comment:
Will have to update this to have the new env vars in https://github.com/0xProject/0x-monorepo/pull/1272